resources/public/contracts/src/Offering.sol

```diff
@@ -176,11 +176,13 @@ contract Offering {
     function doTransferOwnership(address payable _newOwner)
         private
     {
-        ens.setOwner(offering.node, _newOwner);
         if (isNodeTLDOfRegistrar()) {
             uint256 tokenId = uint256(offering.labelHash);
             BaseRegistrar registrar = BaseRegistrar(ens.owner(rootNode));
+            registrar.reclaim(tokenId, _newOwner);
             registrar.transferFrom(registrar.ownerOf(tokenId), _newOwner, tokenId);
+        } else {
+            ens.setOwner(offering.node, _newOwner);
```

Comment:
Because the `registrar.reclaim` function has the nifty option to set the ENS owner to whomever you want if you're token owner (i.e. reclaim to not just yourself). We're using it as you can see, in other words the other branch's `setOwner` is the `reclaim` call.

Reply:
Oh, I see now. Thx